.travis.yml

```diff
@@ -1,8 +1,30 @@
 language: python
-python:
-  - "2.7"
-  - "3.4"
-  - 3.5
+dist: trusty
+sudo: required
 install: pip install -e .
 script: py.test
-sudo: false
+
+python:
+    - "pypy"
+    - "pypy3"
+    - "2.7"
+    - "3.3"
+    - "3.4"
+    - "3.5-dev"
+    - "3.5"
+    - "3.6-dev"
+    - "3.6"
+    - "3.7-dev"
```

Comment:
Let's drop the 3.5/6/7 dev versions. Pickleshare has needed no changes for new Python versions - up to now - for several years. I know that 3.7-dev is also stuck somewhere in the past before the openssl requirement shifted.